tox.ini

```diff
@@ -28,7 +28,10 @@ commands = pytest {posargs}
 basepython = python3.8
 deps =
     flake8==3.8.3
-commands = flake8 --max-complexity 10 src test utils
+    mypy
```

Comment:
please pin the mypy version - I think I saw frequent breaks with mypy updates...

Reply:
Done!